src/EnergyPlus/WeatherManager.cc

```diff
@@ -4232,32 +4232,38 @@ namespace Weather {
             state.dataWeather->WeatherFileExists) {
             if (state.dataWeather->LocationGathered) {
                 // See if "matching" location
-                if (std::abs(state.dataEnvrn->Latitude - state.dataWeather->WeatherFileLatitude) > 1.0 ||
-                    std::abs(state.dataEnvrn->Longitude - state.dataWeather->WeatherFileLongitude) > 1.0 ||
-                    std::abs(state.dataEnvrn->TimeZoneNumber - state.dataWeather->WeatherFileTimeZone) > 0.0 ||
-                    std::abs(state.dataEnvrn->Elevation - state.dataWeather->WeatherFileElevation) / max(state.dataEnvrn->Elevation, 1.0) > 0.10) {
-                    ShowWarningError(state, "Weather file location will be used rather than entered (IDF) Location object.");
-                    ShowContinueError(state, format("..Location object={}", state.dataWeather->LocationTitle));
-                    ShowContinueError(state, format("..Weather File Location={}", state.dataEnvrn->WeatherFileLocationTitle));
-                    ShowContinueError(
-                        state,
-                        format("..due to location differences, Latitude difference=[{:.2R}] degrees, Longitude difference=[{:.2R}] degrees.",
-                               std::abs(state.dataEnvrn->Latitude - state.dataWeather->WeatherFileLatitude),
-                               std::abs(state.dataEnvrn->Longitude - state.dataWeather->WeatherFileLongitude)));
-                    ShowContinueError(state,
-                                      format("..Time Zone difference=[{:.1R}] hour(s), Elevation difference=[{:.2R}] percent, [{:.2R}] meters.",
-                                             std::abs(state.dataEnvrn->TimeZoneNumber - state.dataWeather->WeatherFileTimeZone),
-                                             std::abs((state.dataEnvrn->Elevation - state.dataWeather->WeatherFileElevation) /
-                                                      max(state.dataEnvrn->Elevation, 1.0) * 100.0),
-                                             std::abs(state.dataEnvrn->Elevation - state.dataWeather->WeatherFileElevation)));
+                if (state.dataWeather->keepUserSiteLocationDefinition) {
+                    ShowWarningError(state, "User Entered (IDF) Site:Location object will be used rather than the weather file location.");
```

Comment:
If the user said to use the site location, is a message really needed to verify that choice?

Reply:
Good point.  I didn't see this as "required" but thought it was simply a helpful "confirmation".  Do you want me to take this out?  I don't have a problem removing it.

Reply:
I think I would rather not see a confirmation of an input. I also don't see a change to the eio. Not sure if that's needed but ...

    ! <Site:Location>, Location Name, Latitude {N+/S- Deg}, Longitude {E+/W- Deg},  Time Zone Number {GMT+/-}, Elevation {m},  Standard Pressure at Elevation {Pa}, Standard RhoAir at Elevation

Reply:
In the commit that was just made, I got rid of this confirmation message.  As far as the EIO is concerned, it's going to show the correct location title based on the new field in Site:Location.  That won't show any changes in IDFs that already existed because these all use the default of "No" which means no change from what was done before.  If you change any existing input file to "Yes" you will see a change in the EIO.  So, I don't think there needs to be any change to the code that produces the EIO unless I am missing something or misunderstood your point.

Reply:
Yeah a default of No, to ensure unchanged behavior by default, is good.